---
title: Publications
layout: page
---

My google scholar page is [here](https://scholar.google.com/citations?user=kH44ro8AAAAJ&hl=en).

<h2>Machine Learning and Oncology Radiomics</h2>

Michael Baine, Justin Burr, **Qian Du**, Chi Zhang, Xiaoying Liang, Luke Krajewski, Laura Zima, Gerard Rux, Dandan Zheng.
The Potential Use of Radiomics with Pre-Radiation Therapy MR Imaging in Predicting Risk of Pseudoprogression in Glioblastoma Patients.
*Journal of Imaging*, Feb 2021. [doi.org/10.3390/jimaging7020017](https://doi.org/10.3390/jimaging7020017)

Yu Shi, Ethan Wahle, **Qian Du**, Luke Krajewski, Xiaoying Liang, Sumin Zhou, Chi Zhang, Michael Baine, Dandan Zheng.
Associations between Statin/Omega3 Usage and MRI-Based Radiomics Signatures in Prostate Cancer.
*Diagnostics*, Jan 2021. [doi.org/10.3390/diagnostics11010085](https://doi.org/10.3390/diagnostics11010085)

Elsa Parr [1], **Qian Du** [1], Chi Zhang, Chi Lin, Ahsan Kamal, Josiah McAlister, Xiaoying Liang, 
Kyle Bavitz, Gerard Rux, Michael Hollingsworth, Michael Baine, Dandan Zheng
Radiomics-Based Outcome Prediction for Pancreatic Cancer Following Stereotactic Body Radiotherapy
*Cancers*, April 2020. [doi.org/10.3390/cancers12041051](https://doi.org/10.3390/cancers12041051)

**Qian Du**, Michael Baine, Kyle Bavitz, Josiah McAllister, Xiaoying Liang, Hongfeng Yu, Jeffrey Ryckman, 
Lina Yu, Hengle Jiang, Sumin Zhou, Chi Zhang, Dandan Zheng.
Radiomic feature stability across 4D respiratory phases and its impact on lung tumor prognosis prediction
*PLOS One*, May 2019. [doi.org/10.1371/journal.pone.0216480](https://doi.org/10.1371/journal.pone.0216480)

**Qian Du**, Chi Zhang, Xiaofeng Zhu, Xiaoying Liang, Chi Zhang, Vivek Verma, Kenneth Follet, Shuo Wang, Qiyong Fan, 
Rongtao Ma, Sumin Zhou, Dandan Zheng.
Application of statistical and computational methodology to predict brainstem dosimetry for trigeminal neuralgia stereotactic radiosurgery.
*Medical physics*, May 2018. [doi.org/10.1002/mp.12852](https://doi.org/10.1002/mp.12852)

<h2>Statistical Genetics and Salinity Tolerance</h2>

Malachy T Campbell, **Qian Du**, Kan Liu, Sandeep Sharma, Chi Zhang, Harkamal Walia.
Characterization of the transcriptional divergence between the subspecies of cultivated rice (Oryza sativa).
*BMC genomics*, December 2020. [doi.org/10.1186/s12864-020-06786-6](https://doi.org/10.1186/s12864-020-06786-6)

Gleb Moisseyev, Kiyoul Park, Alix Cui, Daniel Freitas, Divith Rajagopal, Anji Reddy Konda, Madalayne Martin-Olenski, 
Mackenzie Mcham, Kan Liu, **Qian Du**, James C Schnable, Etsuko N Moriyama, Edgar B Cahoon, Chi Zhang.
*Database*, Jan 2020. [doi.org/10.1093/database/baaa038](https://doi.org/10.1093/database/baaa038)

**Qian Du**, Malachy Campbell, Huihui Yu, Kan Liu, Harkamal Walia, Qi Zhang, Chi Zhang.
Network‐based feature selection reveals substructures of gene modules responding to salt stress in rice.
*Plant direct*, August 2019. [doi.org/10.1002/pld3.154](https://doi.org/10.1002/pld3.154)

Laetitia Virlouvet, Thomas J Avenson, **Qian Du**, Chi Zhang, Ning Liu, Michael Fromm, Zoya Avramova, Sabrina E Russo.
Dehydration stress memory: gene networks linked to physiological responses during repeated stresses of Zea mays.
*Frontiers in plant science*, July 2018.[doi.org/10.3389/fpls.2018.01058](https://doi.org/10.3389/fpls.2018.01058)

Malachy T Campbell, **Qian Du**, Kan Liu, Chris J Brien, Bettina Berger, Chi Zhang, Harkamal Walia.
A Comprehensive Image‐based Phenomic Analysis Reveals the Complex Genetic Architecture of Shoot Growth Dynamics in Rice (Oryza sativa).
*The Plant Genome*, July 2017. [doi: 10.3835/plantgenome2016.07.0064](https://doi: 10.3835/plantgenome2016.07.0064)

Malachy T Campbell, Nonoy Bandillo, Fouad Razzaq A Al Shiblawi, Sandeep Sharma, Kan Liu, **Qian Du**, 
Aaron J Schmitz, Chi Zhang, Anne-Aliénor Véry, Aaron J Lorenz, Harkamal Walia.
Allelic variants of OsHKT1; 1 underlie the divergence between indica and japonica subspecies of rice (Oryza sativa) for root sodium content.
*PLoS genetics*, June 2017. [doi.org/10.1371/journal.pgen.1006823](https://doi.org/10.1371/journal.pgen.1006823)

<h2>Statistics, Bioinformatics, NGS</h2>

Huihui Yu, **Qian Du**, Malachy Campbell, Bin Yu, Harkamal Walia, Chi Zhang.
R package: VaSP. *New Phytologist*, Jan 2021.[doi.org/10.1111/nph.17189](https://www.bioconductor.org/packages/devel/bioc/manuals/VaSP/man/VaSP.pdf)

Kan Liu, Shangang Jia, **Qian Du**, Chi Zhang.
NanoAsPipe: A transcriptome analysis and alternative splicing detection pipeline for MinION long-read RNA-seq.
*BIBM*, November 2017. [doi.org/10.1109/BIBM.2017.8217937](https://doi.org/10.1109/BIBM.2017.8217937)

Kan Liu, **Qian Du**, Guodong Ren, Bin Yu, Chi Zhang.
Identification of differential alternative splicing events with an adjusted beta-distribution model.
*2017 IEEE International Conference on Electro Information Technology*, May 2017. [doi.org/10.1109/EIT.2017.8053369](https://doi.org/10.1109/EIT.2017.8053369)

Yongchao Dou, Shengjun Li, Weilong Yang, Kan Liu, Qian Du, Guodong Ren, Bin Yu, Chi Zhang.
Genome-wide discovery of circular RNAs in the leaf and seedling tissues of Arabidopsis thaliana.
*Current genomics*, August 2017. [doi.org/10.2174/1389202918666170307161124](https://doi.org/10.2174/1389202918666170307161124)

<h2>Bioinformatics and Bioengineering</h2>

Zhanqi Wang, Fuxing Zuo, Qing Liu, Xuesheng Wu, **Qian Du**, Yuguo Lei, Zhangmin Wu, Haishuang Lin.
Comparative Study of Human Pluripotent Stem Cell-Derived Endothelial Cells in Hydrogel-Based Culture Systems
*ACS omega*, Mar 2021. [doi.org/10.1021/acsomega.0c06187](https://doi.org/10.1021/acsomega.0c06187)

Haishuang Lin, Xuefeng Qiu, **Qian Du**, Qiang Li, Ou Wang, Leonard Akert, Zhanqi Wang, Dirk Anderson, 
Kan Liu, Linxia Gu, Chi Zhang, Yuguo Lei.
Engineered microenvironment for manufacturing human pluripotent stem cell-derived vascular smooth muscle cells.
*Stem cell reports*, Jan 2019. [doi.org/10.1016/j.stemcr.2018.11.009](https://doi.org/10.1016/j.stemcr.2018.11.009)

Haishuang Lin, **Qian Du**, Qiang Li, Ou Wang, Zhanqi Wang, Kan Liu, Leonard Akert, Chi Zhang, Soonkyu Chung, Bin Duan, Yuguo Lei.
Differentiating human pluripotent stem cells into vascular smooth muscle cells in three dimensional thermoreversible hydrogels.
*Biomaterials science*, [https://doi.org/10.1039/C8BM01128A](doi.org/10.1039/C8BM01128A)

Haishuang Lin, **Qian Du**, Qiang Li, Ou Wang, Zhanqi Wang, Christian Elowsky, Kan Liu, Chi Zhang, 
Soonkyu Chung, Bin Duan, Yuguo Lei.
Manufacturing human pluripotent stem cell derived endothelial cells in scalable and cell-friendly microenvironments.
*Biomaterials science*, [doi.org/10.1039/C8BM01095A](https://doi.org/10.1039/C8BM01095A)

Haishuang Lin, Qiang Li, **Qian Du**, Ou Wang, Zhanqi Wang, Leonard Akert, Mark A Carlson, Chi Zhang, 
Anuradha Subramanian, Matthew Lunning, Ming Li, Yuguo Lei.
*Biomaterials*. Jan 2019. [doi.org/10.1016/j.biomaterials.2018.10.027](https://doi.org/10.1016/j.biomaterials.2018.10.027)

Haishuang Lin, **Qian Du**, Qiang Li, Ou Wang, Zhanqi Wang, Neety Sahu, Christian Elowsky, Kan Liu, Chi Zhang, 
Soonkyu Chung, Bin Duan, Yuguo Lei.
A scalable and efficient bioprocess for manufacturing human pluripotent stem cell-derived endothelial cells.
*Stem cell reports*, August 2018. [doi.org/10.1016/j.stemcr.2018.07.001](https://doi.org/10.1016/j.stemcr.2018.07.001)

Haishuang Lin, **Qian Du**, Qiang Li, Ou Wang, Zhanqi Wang, Kan Liu, Christian Elowsky, Chi Zhang, Yuguo Lei.
Hydrogel-Based Bioprocess for Scalable Manufacturing of Human Pluripotent Stem Cell-Derived Neural Stem Cells.
*ACS applied materials & interfaces*, August 2018. [doi.org/10.1021/acsami.8b05780](https://doi.org/10.1021/acsami.8b05780)

Qiang Li, Haishuang Lin, **Qian Du**, Kan Liu, Ou Wang, Catelyn Evans, Hannah Christian, Chi Zhang, Yuguo Lei.
Scalable and physiologically relevant microenvironments for human pluripotent stem cell expansion and differentiation.
*Biofabrication*, Feb 2018. [doi.org/10.1088/1758-5090/aaa6b5](https://doi.org/10.1088/1758-5090/aaa6b5)







